bld/namelist_files/namelist_defaults_usr_files.xml

```diff
@@ -24,6 +24,7 @@ provided they are in the valid list expressed above.
 
 <!-- Surface dataset (relative to ${csmdata}) -->
 <fsurdat phys="clm4_5">lnd/clm2/surfdata_map/surfdata_${clm_usr_name}_simyr${sim_year}.nc</fsurdat>
+<fsurdat phys="clm5_0">lnd/clm2/surfdata_map/surfdata_${clm_usr_name}_simyr${sim_year}.nc</fsurdat>
```

Comment:
Is this unique to FAN?

Reply:
To me it would seem like a reasonable default regardless of FAN?